### PR TITLE
databases week2 homework

### PR DESCRIPTION
**Pull request by vitalreddym**

### CODE REVIEW

File: databases/week2/School.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
+CREATE DATABASE School
+DEFAULT CHARACTER SET = "utf8mb4";
+USE School;
+
+
+CREATE TABLE Class (
+     id int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
```
Comment (lawansubba) [inline databases/week2/School.sql:7]: good use of PK and auto_increment

File: databases/week2/School.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+     name varchar(255) NOT NULL,
+     begins_date DATE NOT NULL,
+     ends_date DATE NOT NULL
+);
+ALTER TABLE Class ADD status ENUM('not-started', 'ongoing', 'finished');
+
+
+CREATE TABLE Student(
+    id int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
```
Comment (lawansubba) [inline databases/week2/School.sql:16]: good use of PK and auto_increment

File: databases/week2/hyf2.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
+SELECT task.* FROM task JOIN user_task ON user_task.task_id = task.id
```
Comment (lawansubba) [inline databases/week2/hyf2.sql:2]: please indent your code as best practice
SELECT 
	task.* 
FROM 
	task	 
	JOIN user_task ON user_task.task_id = task.id
	JOIN user ON user_task.user_id = user.id 
WHERE 
	user.email LIKE '%spotify.com%';

**Review by lawansubba — COMMENTED** [review]
Good work on part 1,2,3
On part 4 
the Foreign key relationship appears to be missing  and  there is no link between student and professor in your tables so there shouldnt be one in the entity diagram either

File: databases/week2/hyf2.sql (hunk 3 above)
Comment (vitalreddym) [reply databases/week2/hyf2.sql:2]: ok

**Pull request merged**